tests/fixtures/html/00351-v1-vor-section-bib2-bib.html

```diff
@@ -0,0 +1 @@
+<div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/483531a&link_type=DOI" target="_blank"><span class="nlm-article-title">Drug development: raise standards for preclinical cancer research</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:CG Begley&quot;" target="_blank">CG Begley</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:LM Ellis&quot;" target="_blank">LM Ellis</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source">Nature</span></span>, <span class="elife-reflink-details-volume">483</span>, <span class="elife-reflink-details-pages">531-533</span>, <span class="elife-reflink-details-year">2012</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-uri"><a href="http://dx.doi.org/10.1038/483531a" target="_blank">http://dx.doi.org/10.1038/483531a</a></span></div></div></div>
```

Comment:
This will need a little cleanup. It matches the current site but we will not be implementing a lookup service for external links.

```<a href="/lookup/external-ref/doi?access_num=10.1038/483531a&link_type=DOI" target="_blank">```

will have to become:

```<a href="http://dx.doi.org/10.1038/483531a" target="_blank">```